src/coreclr/jit/sideeffects.cpp

```diff
@@ -141,15 +141,9 @@ AliasSet::NodeInfo::NodeInfo(Compiler* compiler, GenTree* node)
     if (node->IsCall())
     {
         // For calls having return buffer, update the local number that is written after this call.
-        GenTree* retBufArgNode = node->AsCall()->GetLclRetBufArgNode();
+        GenTree* retBufArgNode = compiler->gtCallGetDefinedRetBufLclAddr(node->AsCall());
         if (retBufArgNode != nullptr)
         {
-            // If a copy/reload is inserted by LSRA, retrieve the returnBuffer
```

Comment:
Why is this no longer needed? I don't recall which test, but I added it much later when trying to debug a test failure.

Reply:
I have moved it into `gtCallGetDefinedRetBufLclAddr` instead, since it also needs to skip it for the assertion there.